### PR TITLE
Remove goog style

### PR DESCRIPTION
**Pull request by nicholas-l**
Remove goog.style.*

### CODE REVIEW

**Comment by nicholas-l** [comment]
Should be good for review, is anyone able to test this on IE?

**Comment by nicholas-l** [comment]
See #5118 for the other part of classlist removal.

**Comment by nicholas-l** [comment]
ping @ahocevar , are you able to review this?

**Comment by ahocevar** [comment]
Thanks for the work on this @nicholas-l, and sorry this got unnoticed for so long. I should be able to review today or tomorrow.

**Comment by nicholas-l** [comment]
All good :) Can squash/make changes where needed

**Comment by ahocevar** [comment]
Great work @nicholas-l. Please consider cherry-picking f5cf496555e327e3ffc7676e614bfbf34401d3b3, to avoid the creation of object literals whose values are only used once.

File: src/ol/control/attributioncontrol.js [hunk 1]
```diff
@@ -44,7 +43,7 @@ ol.control.Attribution = function(opt_options) {
   this.logoLi_ = document.createElement('LI');
 
   this.ulElement_.appendChild(this.logoLi_);
-  goog.style.setElementShown(this.logoLi_, false);
+  this.logoLi_.style.display = 'none'; // Hide the logo
```
Comment (ahocevar) [inline src/ol/control/attributioncontrol.js]: This `// Hide the logo` comment and similar comments throughout this pull request could be removed, if you get a chance.

File: src/ol/control/zoomslidercontrol.js [hunk 2]
```diff
@@ -185,11 +184,21 @@ ol.control.ZoomSlider.prototype.setMap = function(map) {
...
+  var thumbComputedStyles = window.getComputedStyle(thumb);
+  var thumbMargins = {
+    left: parseFloat(thumbComputedStyles['marginLeft']),
+    right: parseFloat(thumbComputedStyles['marginRight']),
+    top: parseFloat(thumbComputedStyles['marginTop']),
+    bottom: parseFloat(thumbComputedStyles['marginBottom'])
+  };
+  var thumbBorderBoxSize = {
+    width:thumb.offsetWidth, height: thumb.offsetHeight
+  };
```
Comment (ahocevar) [inline src/ol/control/zoomslidercontrol.js]: Like in my suggested commit for `src/ol/map.js`, the `thumbMargins` and `thumbBorderBoxSize` object literals could be avoided.

**Comment by probins** [comment]
> cherry-picking beaedbc

but we got rid of (nearly all) `goog.global` ?

**Comment by ahocevar** [comment]
> but we got rid of (nearly all) goog.global ?

Did we?

File: src/ol/overlay.js [hunk 3]
```diff
@@ -440,12 +439,14 @@ ol.Overlay.prototype.getRect_ = function(element, size) {
   goog.asserts.assert(element, 'element should be defined');
   goog.asserts.assert(size !== undefined, 'size should be defined');
 
-  var offset = goog.style.getPageOffset(element);
+  var box = element.getBoundingClientRect();
+  var offsetX = box.left + window.pageXOffset;
```
Comment (ahocevar) [inline src/ol/overlay.js]: Should be `goog.global` for now (also on next line).

**Comment by ahocevar** [comment]
Ah, you're right @probins - `ol.global` is what we use now.

**Comment by nicholas-l** [comment]
Did you want me to squash it down again?

**Comment by ahocevar** [comment]
I think we're good. The distinct commits make sense. Thanks a lot for your work on this @nicholas-l.

**Comment by fredj** [comment]
Thanks @nicholas-l

**Pull request merged**